### PR TITLE
feat: expose arguments `cpu`, `num_workers` and `queue` in `finetuner.fit`

### LINKED ISSUE #409: Enable GPU runs
We should allow the use of GPUs and the option to specify a number of CPUs in the client. API already supports this

### PR DESCRIPTION
**Pull request by gmastrapas**
Closes #409

### CODE REVIEW

**Review by bwanglzu — CHANGES_REQUESTED** [review]
can we remove `queue`?

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- Expose arguments `cpu`, `num_workers` and `queue` in `finetuner.fit`. ([#411](https://github.com/jina-ai/finetuner/pull/411))
```
Comment (jupyterjazz) [inline CHANGELOG.md:13]: Can you update this line as well?

File: tests/unit/test_client.py [hunk 2]
```diff
@@ -61,6 +61,9 @@ def test_create_run(client_mocker, experiment_name='exp', run_name='run'):
         experiment_name=experiment_name,
         run_name=run_name,
         run_config=config,
+        device='cpu',
+        cpus=1,
+        gpus=1,
```
Comment (jupyterjazz) [inline tests/unit/test_client.py:66]: Can you also check that they were sent correctly? i.e. `assert sent_request['json_data'][GPUS] == gpus`

File: finetuner/experiment.py [hunk 3]
```diff
@@ -152,10 +155,17 @@ def create_run(
...
+        cpu = kwargs.get(CPU, True)
+        num_workers = kwargs.get(NUM_WORKERS, 4)
+
         run_info = self._client.create_run(
             run_name=run_name,
             experiment_name=self._name,
             run_config=config,
+            device='cpu' if cpu else 'gpu',
+            cpus=num_workers,
+            gpus=1,
```
Comment (jupyterjazz) [inline finetuner/experiment.py:168]: Why should we bother having `gpus` here if it's always 1?

**Review by bwanglzu — COMMENTED** [review]
why did we remove `wandb` option? it's not working?

File: finetuner/experiment.py (hunk 3 above)
Comment (gmastrapas) [reply finetuner/experiment.py:168]: At some point it will be used

**Comment by gmastrapas** [comment]
> why did we remove `wandb` option? it's not working?

I'd rather not prompt users to give us their wandb tokens. We need to find a better way for this

**Review by bwanglzu — APPROVED** [review]
LGTM! in the finetuner runner, we need to parse these two parameters as well.

**Review by jupyterjazz — APPROVED** [review]
:rocket:

**Pull request merged**